### DISCUSSION TITLE
新しいアイコンの提案です。

### DISCUSSION THREAD

**Discussion by Mattz-P** [limura/NovelSpeaker#128] [Ideas]
# 提案＃１

* Loading
![Loading](https://user-images.githubusercontent.com/33806410/106781490-fa92fb80-65fd-11eb-8a41-5e193b9ea88b.png)
* NoBorder
![Noborder](https://user-images.githubusercontent.com/33806410/106781499-fcf55580-65fd-11eb-85e9-b83af24b1964.png)
* NovelSpeaker1
![NovelSpeaker1](https://user-images.githubusercontent.com/33806410/106781501-fd8dec00-65fd-11eb-9a04-34ceb207459a.png)

# 提案＃２

* Loading2
* ![Loading2](https://user-images.githubusercontent.com/33806410/106787027-0a154300-6604-11eb-838a-1ee3bb3f8c15.png)
* NoBorder2
![Noborder2](https://user-images.githubusercontent.com/33806410/106788182-7cd2ee00-6605-11eb-9bb7-0d7e6738d565.png)
* NovelSpeaker2
![NovelSpeaker2](https://user-images.githubusercontent.com/33806410/106788501-e2bf7580-6605-11eb-9d88-9f04a7bf36ad.png)

# ベクトル原画 (Illustrator files .ai)
[NovelSpeaker.zip](https://github.com/limura/NovelSpeaker/files/5920426/NovelSpeaker.zip)

**Comment by limura** [comment]
うわー！
素晴らしい ！ありがとうございます！

今使っているアイコンは「アイコンが無いのはよくないけれど自分で作るのも何だよなぁ」と思いながら、クラウドソーシングサービスにでも依頼しようかと思って、「依頼するなら少しはどんな物が欲しいのかのサンプル的な物があったほうがいいかなぁ」と思い直してフリー素材を探して貼り付けて作った、という程度の物なので、こういう感じで綺麗に作り直して頂けるのはとても嬉しいです。ありがとうございます。

特に問題がなければそのまま取り込めるように加工するなどしてしまおうかと思うのですが、問題ありませんでしょうか？

ただ、今現在はお問い合わせ対応に時間が取られているというのと、致命的な問題が発生している方がおられるらしいので、そちらの対応を先にしてからの作業になりそうです。そのため、アイコンを適用する作業はすぐには完了しなさそうな事はご了承いただけますと幸いです。

**Reply by Mattz-P** [reply]
喜んでくれて嬉しいです。
`特に問題がなければそのまま取り込めるように加工するなどしてしまおうかと思うのですが、問題ありませんでしょうか？`
了解しました。特に問題ありません。どうぞお好きなように加工または調整してください。
開発における問題によりアイコンのアップデートの遅延も了解いたしました。開発を頑張ってください！

**Reply by limura** [reply]
ありがとうございます！

提案1(本が閉じている方) と 提案2(本が開いている方) は甲乙つけ難く、どちらを選ぼうかと悩んでいます。
どちらかオススメなものはありますか？

デザイン的な知識がまるでありませんもので、感想のような事しか言えないのですけれど、

提案1(本が閉じている方) はメリハリが効いている感じで印象に残りやすい気がしていて、
提案2(本が開いている方) は本側の枠線の太さとヘッドホン側の太さの調和が取れている感じがしています。

うーん……悩ましい……

**Reply by Mattz-P** [reply]
デザインの評価ありがとうございました！
個人的に提案2が好きですが、デザイン中の「本」自体はフリー素材です。そのため一部が他のアプリとカブる（被るか冠る；漢字の区別ができません(´・ω・`)）可能性があります。その代わり、提案一はオリジナルですが、自分が雑にベクトルペンで書いたものの、全体的のバランスが少し崩したデザインです。
私が適当に選ぶなら2ですかね('ω')。

**Reply by limura** [reply]
ありがとうございます！
それでは、提案2 の方を使わせていただきます。

ところで、Mattz-P様の権利表記をアプリ内の「設定タブ」→「権利表記」の所に「アイコン作成」の項目で追加したいと思うのですが、どうお書きすると良いでしょうか。
また、場合によっては 提案2 の中のフリー素材の方の側の権利表記も必要かもしれませんのでその辺りも教えて頂けますとありがたいです。

P.S. 「かぶる」はこの場合は多分「被る」でも「冠る」でもよさそうな気がします。自分の場合は「被る」と書きそうです。
あと、オンライン辞書を一つ引いてみました所、どちらを使ってもよさそうでした。
https://dictionary.goo.ne.jp/word/%E8%A2%AB%E3%82%8B_%28%E3%81%8B%E3%81%B6%E3%82%8B%29/

**Reply by Mattz-P** [reply]
`どうお書きすると良いでしょうか。`
権利表記ありがとうございます！Mattz-Pでいいと思います。
提案2の中、本のフリー素材 SHARED BY: [MOHAMED IBRAHIM](http://www.clker.com/profile-9.html) 04-05-2008
あと、わざわざ被るの漢字説明してくれてありがとうございますｗ(*‘ω‘ *)

**Reply by limura** [reply]
ありがとうございます。
それでは、

--- NovelSpeaker アイコン ---

作成者: Mattz-P @ GitHub (https://github.com/Mattz-P)

アイコン内素材:
SHARED BY: MOHAMED IBRAHIM 04-05-2008 (http://www.clker.com/profile-9.html)


という表記にさせていただきます。
問題がありそうなら教えて下さい。

それでは、ありがたく本体への組み込み作業に入りますね。
今の所 IceCream brunch で作業しているのですが、
こちらのアイコンの組み込み作業が終わりました辺りで IceCream brunch を master brunch に merge して、そのままリリースの準備に入ろうと思っています。

**Reply by Mattz-P** [reply]
了解しました。時に問題ありません。ご武運を祈ります！(*‘ω‘ *)

**Reply by limura** [reply]
ありがとうございます！
早めにリリースできるようがんばりまーす！( ﾟ∀ﾟ)b
